### DISCUSSION TITLE
Migrating from vifm, 2 pane workflow

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [sxyazi/yazi#429] [1. Q&A]
What is the recommended way to move stuff from one directory to another?

In vifm , there are two panes and i just swap between them and move/copy from one to another.

**Comment by sxyazi** [comment]
I haven't used vifm, but Yazi's multi-tabs seem to achieve similar functionality - you can copy, move, paste across tabs, and quickly switch between different tabs using `1`, `2`, `3`, etc., keys.

**Comment by sxyazi** [comment]
For dual-pane, tracked by https://github.com/yazi-rs/plugins/issues/9 now
